extensions/chatgpt-quick-actions/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # ChatGPT Quick Actions Changelog
 
+## [Add Transform and Preview Transform commands] - 2024-09-12
```

Comment:
```suggestion
## [Add Transform and Preview Transform commands] - {PR_MERGE_DATE}
```

Reply:
@pernielsentikaer I switched to {PR_MERGE_DATE} as requested. is it documented somewhere? I can't find any documentation for it.

Reply:
@JonathanAquino Me neither but it was the same [here](https://github.com/raycast/extensions/pull/13789/commits/2566c32c5b4d15647769cea994aab50daab42f87) in my recent contribution.

Reply:
I don't think it's documented, will make sure it will be 🙂

Reply:
@pernielsentikaer it's weird - you merged https://github.com/raycast/extensions/pull/8228 on Sept 4 which documents PR_MERGE_DATE, but the change seems to have gotten reverted in https://github.com/raycast/extensions/commit/24f7f25ccadbdfcc4fd42961d4af8622a02fd9a1 on Sept 6.